src/lucky/params.cr

```diff
@@ -393,24 +473,12 @@ class Lucky::Params
     request.headers["Content-Type"]?
   end
 
-  @_parsed_json : JSON::Any?
-
-  private def parsed_json : JSON::Any
-    @_parsed_json ||= begin
-      if body.blank?
-        JSON.parse("{}")
-      else
-        JSON.parse(body)
-      end
-    end
-  rescue JSON::ParseException
-    raise Lucky::ParamParsingError.new(request)
+  private memoize def parsed_json : JSON::Any
```

Comment:
Does this work? 😂  I didn't know that!

Reply:
Yup! It looks weird but macros before defs have to go between the private haha